File: Nova pasta/entregavel 2 java/contagem.java
import java.util.Scanner;

public class Contagem {


    public void contagem(){
        Scanner input = new Scanner(System.in);
        System.out.println("Número de notas");
        byte n = input.nextByte();
        System.out.println("Nota do aluno:");
        byte nota = input.nextByte();

        byte contagem = 0;
        byte i = 0;
        while (i < n) {
            i++;

            if (nota >= 0 && nota <= 10) {
                contagem++;
            }
        }
        System.out.println("Número de aprovações: " + contagem);
        input.close();
    }












}

File: Nova pasta/entregavel 1 js/contagem.js
var n = prompt("Número de notas");
var nota;
var contagem = 0;
var i = 0;
while (i < n) {
i = i + 1;
nota = prompt("Nota do aluno:");
if (nota >= 7 && nota <= 10) {
contagem = contagem + 1;
}
}
document.write("Número de aprovações: "
+ contagem);
